packages/vc-http-api-test-server/__fixtures__/credentials/index.js

```diff
@@ -0,0 +1,38 @@
+module.exports = [
+    {
+        name: "USCIS Permanent Residency Card",
+        data: require('./case-1.json')
```

Comment:
nit: consider recursive load from filesystem, so that new stuff does not need to be manually registered.